.nuke/parameters.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "./build.schema.json",
+  "Solution": "Petabridge.App.sln"
```

Comment:
This will break whenever anyone uses the template as the name is set via the `dotnet new --name` parameter

Reply:
Good catch. Will fix this and the rest of the templates project. Thanks